src/main/java/medicaldoctor/controllers/navbar/ProfileController.java

```diff
@@ -0,0 +1,86 @@
+package medicaldoctor.controllers.navbar;
+
+import medicaldoctor.entities.UserInformationGenerator;
+import java.net.URL;
+import java.util.HashSet;
+import java.util.ResourceBundle;
+import javafx.event.ActionEvent;
+import javafx.fxml.FXML;
+import javafx.fxml.Initializable;
+import javafx.scene.control.Button;
+import javafx.scene.control.ChoiceBox;
+import javafx.scene.control.Label;
+import javafx.scene.control.TextField;
+import medicaldoctor.controllers.ControllerManager;
+import medicaldoctor.controllers.ParentController;
+import medicaldoctor.core.AppSession;
+import static medicaldoctor.core.AppSession.ENCRYPTION;
+import medicaldoctor.entities.User;
+
+public class ProfileController implements Initializable, ParentController{
```

Comment:
with static controller manager, you dont need the ParentController interface

Reply:
We don't have a lot of time. If I spend all my time making fixes like this we won't get anything done. This works so please leave it

Reply:
I agree, but I am providing comments so they can be changed when first time implementing other controllers.

Reply:
We can learn how to fix things for the future so we aren't continually producing a messy change.